ocaml/testsuite/tests/typing-modes/portable-contend.ml

```diff
@@ -224,12 +224,12 @@ Error: This expression has type bytes but an expression was expected of type
 |}]
 
 let foo : 'a @ contended portable -> (string -> string) @ portable @@ nonportable contended = fun a b -> best_bytes ()
-(* CR layouts v2.8: arrows should cross contention. *)
 [%%expect{|
-Line 1, characters 4-118:
+Line 1, characters 105-118:
 1 | let foo : 'a @ contended portable -> (string -> string) @ portable @@ nonportable contended = fun a b -> best_bytes ()
-        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Error: This value is contended but expected to be uncontended.
+                                                                                                             ^^^^^^^^^^^^^
+Error: This expression has type bytes but an expression was expected of type
```

Comment:
Is this what this test was expecting to test? @riaqn

Reply:
The typo is mine - could you fix this so this test passes?

Reply:
I fixed the typo here, as well as two other places (immediately before and after this test).